.github/workflows/ci-build.yml

```
@@ -16,37 +16,26 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 10.x
-      - name: install dependencies
-        run: yarn install --frozen-lockfile --non-interactive
-      - name: lint
-        run: yarn lint
-      - name: test
-        run: yarn test:ember
-      - name: test node
-        run: yarn test:node
+      - uses: volta-cli/action@v1
+      - run: yarn install --frozen-lockfile
+      - run: yarn lint
+      - run: yarn test:ember
+      - run: yarn test:node
 
   floating-dependencies:
     name: Floating dependencies
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 10.x
-      - name: install dependencies
-        run: yarn install --no-lockfile --non-interactive
-      - name: test
-        run: yarn test:ember-compatibility
+      - uses: volta-cli/action@v1
+      - run: yarn install --no-lockfile
+      - run: yarn test:ember-compatibility
```

```suggestion
      - run: yarn test:ember
      - run: yarn test:node
```

We shouldn't run all Ember version compat checks (those are already ran with `--no-lockfile` in their individual jobs)